service/controller/network/v1/key/key.go

```diff
@@ -0,0 +1,27 @@
+package key
+
+import (
+	"github.com/giantswarm/apiextensions/pkg/apis/core/v1alpha1"
+	"github.com/giantswarm/microerror"
+)
+
+// ToCustomObject converts value to v1alpha1.ClusterNetworkConfig and returns it or
+// error if type does not match.
+func ToCustomObject(v interface{}) (v1alpha1.ClusterNetworkConfig, error) {
+	customObjectPointer, ok := v.(*v1alpha1.ClusterNetworkConfig)
+	if !ok {
+		return v1alpha1.ClusterNetworkConfig{}, microerror.Maskf(wrongTypeError, "expected '%T', got '%T'", &v1alpha1.ClusterNetworkConfig{}, v)
+	}
+
+	if customObjectPointer == nil {
+		return v1alpha1.ClusterNetworkConfig{}, microerror.Maskf(emptyValueError,
```

Comment:
IMO we should not break lines for these things. For me personally this makes readability just worse.